supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchBackendInitializationData.scala

```diff
@@ -43,4 +43,7 @@ case class AwsBatchBackendInitializationData
   // TODO: We'll need something specific for batch probably, but I need to
   //       understand more about the genomics node first
   //genomics: Genomics
-) extends StandardInitializationData(workflowPaths, runtimeAttributesBuilder, classOf[AwsBatchExpressionFunctions])
+) extends StandardInitializationData(workflowPaths, runtimeAttributesBuilder,
```

Comment:
Could this logic be wrapped into a function? It's just a little weird to see a `match` clause appear in an `extends X(...)`line

Reply:
Done.